msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1455,6 +1455,13 @@ When you use these install and uninstall events, there are some instances where
 
 It's important to know that new events can be added anytime in the future and your bot begins to receive them. So you must design for the possibility of receiving unexpected events. If you're using the Bot Framework SDK, your bot automatically responds with a 200 – OK to any events you don't choose to handle.
 
+## Handling errors in conversation events
+
+When a bot encounters an error while handling different events or activities, the bot must send relevant messages in conversations based on the specific events. For example, Welcome message must be handled through [Installation update event](#installation-update-event).
+
+> [!NOTE]
+> In production environment, you must consider Azure Application Insights for events or errors logging. For more information, see [https://aka.ms/bottelemetry](https://aka.ms/bottelemetry) to add telemetry capture to your bot.
```

Comment:
Could you please update it like below:

While in development phase, it's always helpful to send the meaningful messages in conversations which provide more details about specific error for better debugging. However In production environment, you should consider logging this to Azure Application Insights. Visit https://aka.ms/bottelemetry to see how to add telemetry capture to your bot.

Reply:
Hello @aosolis - Could you please confirm if this looks fine to you?

Reply:
Updated as per Wajeed's confirmation.